source/ptreferential/ptreferential_ng.cpp

```diff
@@ -436,6 +444,13 @@ std::string make_request(const Type_e requested_type,
             }
             break;
         case Type_e::VehicleJourney:
+            if (current_datetime != boost::none) {
+                res = "(" + res + ") AND " + navitia::type::static_data::get()->captionByType(requested_type);
+                res = res + ".has_active(" + to_iso_string(*current_datetime) + "Z";
+                res = res + ", " + rt_level_to_string(rt_level);
+                res = res + ")";
+                break;
+            }
```

Comment:
```suggestion
            }    
            break;
```
Unless you *do* want to execute the case `Type_e::Impact` when `current_datetime == boost::none`

Reply:
No, Only vehicle_journey

Reply:
I think we better not use break as this case will only applied for vehicle_journey and with current_datetime, non?